external-crates/move/crates/move-cli/src/sandbox/utils/on_disk_state_view.rs

```diff
@@ -411,7 +411,6 @@ impl OnDiskStateView {
             "Found circular dependencies during dependency generation."
         );
         let linkage_context = LinkageContext::new(
-            *package_address,
```

Comment:
Don't we need to add this to `all_dependencies`?

Reply:
We do! -- it's getting added just below this

```rust
        let linkage_context = LinkageContext::new(
            all_dependencies
                .into_iter()
                .map(|id| (id, id))
                .chain(vec![(*package_address, *package_address)])
                .collect(),
        );

```